00_download_data/2_explore_data.ipynb

```diff
@@ -0,0 +1,854 @@
ro21MLJPJ1I591WncuDE6deqENm3aVMiEatSoEaZMmYLRo0fD1tYWLi4uCAgIwOPHj9GzZ08MHDgQn376KebPn8/v07ZtW8ybNw/Tpk2DTCaDtbU1fvjhBzg4OOh8be+//z7mzZuHoUOHwtnZme8bHx8fhIaGYujQobC0tORLFjx+/Bienp4YMmQIJk2ahNWrV2vd7p133sH48eP5sgicvn37qv0sYWtrW+lnBn2ue9y4cUhPT8f//d//QSAQwM3Njf99WLFiBWbOnImtW7fC09NT6wIvXLmN0tJSvPbaa0r315gxY3D48GGlAuuK5s6di/nz5yM8PByMMbz77rvo1KmTxtdbMStI1fjx45GSkoIxY8aAMYbu3btj0qRJMDMz03id2jRt2lTp/nVycsKYMWOwaNGiCsF4dZ8xQ0JC8M4772Dy5MkQCASwt7fH2rVr1U5BJoYhYLqErwmpIh8fH35VtaoaPXo0pk+frvYbPVJ3MMbw008/ISUlBQsWLDB1c4xCKpXik08+4eeT10eXL1/G7NmzcfDgwXrxZp2UlIRJkybh6NGjfBYWN21h7ty5Jm4dIYRUjyH+np04cQI//PCD0nQrUjdlZ2dj7Nix+PPPP9Wu1lwfbd++Ha6urujTp4+pm1IjpFIppk2bhpEjR9aLeo5yuRwLFy6Eu7u70orVhviMSQyDpryRWksqlWLQoEEQCARql4YldUv//v1x8uRJfPjhh6ZuitHcv39fqQh1ffP555/jo48+QmRkZL0IJq1evRoTJkzAnDlzKizlTAghBNi8eTPmzp2LsWPHmroppJq2b9+OYcOG4a233mowwSSgfGpcz549Td2MGnHv3j307NkTTk5OGDJkiKmbU22FhYUICgpCWloaXn/9dVM3h2hAGUqEEEIIIYQQQgghRC+UoUQIIYQQQgghhBBC9EIBJUIIIYQQQgghhBCiFwooEUIIIYQQQgghhBC9mJu6AfqSy+WQyWq+7JNQKDDKeeoi6hvNqG80o77RjPpGM+obzep731hYCE3dBKLCWGMwY6nvv0O1CfW18VBfGw/1tfFQXxuPvuOvOhdQkskYcnOLa/w8IpGtUc5TF1HfaEZ9oxn1jWbUN5pR32hW3/umaVMHUzeBqDDWGMxY6vvvUG1CfW081NfGQ31tPNTXxqPv+IumvBFCCCGEEEIIIYQQvVBAiRBCCCGEEEIIIYTohQJKhBBCCCGEEEIIIUQvda6GEiGEEFJfyGRS5ORkQiot03vf9HQBGKv7BSrNzS3h5NQUQiENSQghhBBiHNUZg9UHhhp/0eiNEEIIMZGcnExYW9vCzs4VAoFAr32FQjPIZPIaaplxMMZQVJSPnJxMNGniZurmEEIIIaSBqM4YrK4z5PiLprwRQgghJiKVlsHOrlGDG8hwBAIB7OwaNdhvBwkhhBBiGg15DGbI8RcFlAghhBATaogDGUUN/foJIYQQYhoNeQxiqGunKW+EEEIIwebNG7FjxxZs374fVlZWAIB9+3bjn3+OQCAQQCaT4p13/oeAgK4AgDNnorBjxxYwxiAWi/Hqq5PQr98AU14CIYQQQkidU5fHYBRQIoQQQgiOHTuK/v0H4cSJfzBsWDiOH/8bFy7EYfXq9TA3N0dqagqmTZuCX3/9E0+ePMb27X9h6dJVsLW1RV5eLt599z/w8mqNVq1am/pSCCGEEELqjLo8BqMpb4QQQkgDFx9/Ee7uzTFq1MvYvXsHgPJvxl5//T8wNy//7snd3QO//fYnRCIRDhzYi3HjJsDW1hYA4OgowoYNv8PLq5XJroEQQgghpK6p62MwCigRQgghdUjUlRR8vC4Gp+KTDXbMgwf3ITx8FDw9vWBhYYGEhBt49iwT7u7NlbZzdBQBwPPnPJSea9SoYRa2JIQQQkjDwI3Boq6kGOyYdX0MRgElQgghpA45EPMIOQVi7It+aJDj5efn4/z5GOzYsQUfffQ+iooKsXv3Nri6uiEj46nStv/+G4usrGdwcXFDRka60nPXrl1BcvITg7SJEEIIIaS24cZgB2IeGeR49WEMRgElQgghpA4JD/aCk4MVIkIMk9r8zz+HMWJEBFauXIcVK9Zgw4bf8e+/cRg8eBg2bvwFUqkUAJCU9BjffLMIZmZmGD48HFu2bEJJSQkAICcnG199tRClpaUGaRMhhBBCSG3DjcHCg70Mcrz6MAajotyEEEJIHRLq74FQfw8IhWaQyeTVPt6BA/swZ85C/mdra2v07RuGjIx0+Pp2wv/+9zYsLCwgl8swZ85CODk5w8nJGSNHjsaMGVNhbm4OsbgU//3vVLRt267a7SGEEEIIqY24MZih1IcxmIAxxkxy5iqSSGTIzS2u8fOIRLZGOU9dRH2jGfWNZtQ3mlHfaFbf++bp08dwdW1ZpX0NFVCqDdT1Q9OmDiZqDdHEWGMwY6nvf19qE+pr46G+Nh7qa+Opib6uzhisvjDE+IumvBFCCCGEEEIIIYQQvVBAiRBCCCGEEEIIIYTohWooEdLA5Z6OQvbBfXAeEQFR31BTN4cQQgghpMGKTonDkUfHYQYzZItz+Mcn+LyMEI8gE7aMEEIqogwlQhq47IP7IM3JQfbBfaZuCiGEEEJIg3bk0XHkivOUgkkAsPXObhO1iBBCNKOAEiENnPOICJg7OcF5RISpm0IIIYQQ0qAN9Rqg9nEGhuiUOCO3hhBCtKMpb4Q0cKK+oTTVjRBCCCGkFgjxCMK93Ae4kH65wnN77x2iaW+EkFqFAkqEEEJIAxYffxFz586El1crAIBUKsXMmXOxefNG9O8/CD169FLaPj39KdauXYWcnGyIxWL4+HTAhx9+DAsLCwBAQsINTJ36Ntav/wUdOvga/XoIIaSue9N3gtqAUomsFNEpcRRUIqSeqA9jMJryRgghhDRwgYFdsXbtBqxduwGTJ0/BunWr1G4nk8kwc+bHGD/+NaxduwE//fQ7zM3N8fPPP/DbHDy4F+PHv4bdu3cYqfWEEFL
```

Comment:
Do you have an idea for what the two clusters of IDH1 mutant cancers might be?

Reply:
Good question! I don't know a lot about glioma, honestly, but looking at Figure 1 of [this paper](https://doi.org/10.1007/s13311-017-0519-x) it seems like there are two major subclasses of IDH-mutant gliomas (oligodendrogliomas and diffuse astrocytomas).

IDH-mutant diffuse astrocytomas seem to be characterized by TP53 and ATRX mutations, so I tried looking at those, to see if they would separate nicely:

![image](https://user-images.githubusercontent.com/2345877/101957647-d1db2080-3bcf-11eb-8f62-43a32c64bc95.png)
![image](https://user-images.githubusercontent.com/2345877/101957600-ba9c3300-3bcf-11eb-94cb-b06b451a2963.png)

So in the methylation data there's not much of a separation, but in the gene expression data lots of samples in the upper cluster seem to have IDH1/TP53/ATRX mutations, so my guess is that those are astrocytomas and the other IDH1 mutant gene expression cluster is oligodendrogliomas. TCGA might have this subtype information (I'm not sure off the top of my head) so it could be interesting to look at that as well!